PROJECT_PROPOSAL.md

```diff
@@ -0,0 +1,118 @@
+
+```
+Start with:
+
+(a+b)(a+b) = c² + 2ab
+
+Expand (a+b)(a+b):
+
+a² + 2ab + b² = c² + 2ab
+
+Subtract "2ab" from both sides:
+
+   a² + b² = c²
+
+...
+```
+
+
+The study of algebraic systems has exploded in breadth and scope over the past century. A huge variety of systems have been proposed and investigated; these are in daily use across mathematics, physics, and computer science. Interest in "mathematical foundations" -- an algebraic system encompassing all of mathematics -- has remained keen since the notion was first conceived, despite its being proved impossible.
+
+In the past few decades, uptake of computer-checked "formal proof assistants" has grown. These are software systems that allow users to construct computer-checked algebraic proofs. Large bodies of work have been successfully formalized and checked using these systems.
+
+The public has no idea that any of this is going on. In fact they have little understanding of what mathematicians do at all. There *is* public interest in mathematics, as demonstrated by the popularity of events like the "Summers of Math Exposition" on YouTube. However, most of the algebraic infrastructure of contemporary mathematics remains out-of-reach except to experts.
+
+This might be surprising to people working in these fields. Contemporary algebraists -- and the school studying "higher category theory" especially -- tend to base their systems on simple and intuitive spatial metaphors. For example:
+
+- Rearranging text (traditional algebra)
+- Gluing together triangles (commutative diagrams)
+- Sliding around beads on tangled wires (string diagrams)
+
+And so on. How could anybody not understand these ideas?
+
+Part of the issue is that the literature is dense and abstract. There's a saying:
+
+> "Trying to learn math by reading only published papers is like trying to learn chess by reading only chess notation."
+
+Published proofs tend to convey exactly enough information to convince a *specialist* that an argument works. A non-specialist, however, will have absolutely no idea what's going on.
+
+This applies twice over to computer-checked proofs. Proof systems are targeted towards expert users, who are *already familiar* with various complicated rulesets (software development tools, dependent type systems, higher-order unification, etc.) A proof written with such an assistant is very convincing *to the person who wrote it*, and maybe her friends, but to non-experts it is a black box. 
+
+Other mathematicians may be convinced *that* the proof is correct, simply by its being written in a proof assistant. Surprisingly, though, looking at its source code will not tell them much about how it works, at least without in-depth study. This makes it harder to establish trust within the mathematical community. Results that have only been checked by computer, such as the Four-Color theorem, are widely mistrusted, and much effort has been made to find fully human-checkable proofs of the same.
```

Comment:
I'm really not sure about this paragraph. I think it's asserting too much.

All I'm really trying to say is that I have trouble understanding proof assistant proofs :)